recipes/MADL-CPU-IntelMPI/README.md

```diff
@@ -0,0 +1,59 @@
+# MADL-CPU-OpenMPI
+This recipe shows how to run Microsoft Azure Distributed Linear Learner (MADL) on CPUs across
+Azure VMs via Open MPI.
+
+## Configuration
+Please see refer to this [set of sample configuration files](./config) for
+this recipe.
+
+### Pool Configuration
+The pool configuration should enable the following properties:
+* `vm_size` should be a CPU-only instance, 'STANDARD_D2_V2'.
```

Comment:
Best to leave out the size? Users can select any size?

Reply:
I haven't tested this deployment on other sizes, or on RDMA enabled instances. The only requirement is the VM has to be CPU.

How about this: `vm_size` should be a CPU-only non RDMA instance.

Reply:
I just saw that you are reviewing the files under recipes/MADL-CPU-IntelMPI/ and not under recipes/MADL-CPU-OpenMPI/
I have deleted all the files under the IntelMPI.  The dir won't be there once the PR is approved.